### PR TITLE
test: ✅ Enable transition and animation in jest

### PR DESCRIPTION
**Pull request by afc163**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [x] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution

解决 Form 和 Alert 下三行动画相关代码测试未覆盖的问题。

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |     -      |
| 🇨🇳 Chinese |    -       |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
node 下的用例出错：https://app.circleci.com/pipelines/github/ant-design/ant-design/16937/workflows/c6c1666d-16cd-4bec-962b-07f9afc21ba8/jobs/229847

这里和 #26519 是同一个问题，rc-motion 的服务端渲染问题。cc @zombieJ 

image

File: tests/setup.js [hunk 1]
```diff
@@ -21,6 +21,11 @@ if (typeof window !== 'undefined') {
       })),
     });
   }
+
+  // Fix css-animation deps on these
+  // https://github.com/yiminghe/css-animation/blob/a5986d73fd7dfce75665337f39b91483d63a4c8c/src/Event.js#L44
+  window.AnimationEvent = window.AnimationEvent || (() => {});
+  window.TransitionEvent = window.TransitionEvent || (() => {});
```
Comment (afc163) [inline tests/setup.js:28]: 之前没有开启这个，测试不出来。

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
只剩 6 行代码没覆盖了！

**Pull request merged**